src/pybind/mgr/dashboard/frontend/src/app/ceph/shared/smart-list/smart-list.component.html

```diff
@@ -7,60 +7,79 @@
                   i18n>The data received has the JSON format version 2.x and is currently incompatible with the dashboard.</cd-alert-panel>
 
   <ng-container *ngIf="!error && !incompatible">
-    <cd-alert-panel *ngIf="!(data | keyvalue).length"
+    <cd-alert-panel *ngIf="isEmpty(data)"
```

Comment:
Isn't it better to use a pipe here? I know it's too late for that because this is a backport PR. I will try to improve that in master and backport it to octopus.

* https://tracker.ceph.com/issues/42908
* https://blog.appverse.io/why-it-is-a-bad-idea-to-use-methods-in-the-html-templates-with-angular-2-30d49f0d3b16